src/App.js

```diff
@@ -30,7 +37,37 @@ function App() {
   return (
     <div className="page">
       <div className="content">
-        {shownScreen}
+        <Router>
+          <div>
+            <nav>
+              <ul>
+                <li>
+                  <Link to="/">Home</Link>
+                </li>
+                <li>
+                  <Link to="/signUp">Register</Link>
+                </li>
+                <li>
+                  <Link to="/forgotPassword">Forgot Password</Link>
+                </li>
+              </ul>
+            </nav>
```

Comment:
The sign up and sign in page probably wont have this nav bar. I'd move that inside a component and mount those on the pages that need a nav bar

Reply:
nav bar has been removed